addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/PathTraversalScanRule.java

```diff
@@ -387,8 +406,6 @@ public void scan(HttpMessage msg, String param, String value) {
 
             // Check 3: Detect if this page is a directory browsing component
             // example: https://www.buggedsite.org/log/index.php?dir=C:\
-            // note that depending on the AttackLevel, the number of prefixes that we will try
-            // changes.
```

Comment:
I think this comment could just be removed, it seems clear just looking at the code.

Reply:
I don't get your point. The lines have been removed, so what to remove else?

Reply:
The whole comment not just half.

Reply:
The example could go. I think the first line (check 3….) is worth keeping.

(At least until the 5 checks are broken into discrete methods…not necessarily for this PR)

Reply:
The example here should still be removed

Reply:
done